Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -271,82 +271,29 @@ function New-LDAPIdentitySource {
     }
 }
 
-<#
-    .Synopsis
-     Download the certificate from a domain controller
-#>
-function Get-CertificateFromDomainController {
-    param (
-        [Parameter(
-            Mandatory = $true)]
-        [ValidateNotNull()]
-        [System.Uri]
-        $ParsedUrl,
-
-        [Parameter(
-            Mandatory = $true)]
-        [ValidateNotNull()]
-        [string]
-        $computerUrl
-    )
-
-    try {
-        $Command = 'nslookup ' + $ParsedUrl.Host + ' -type=soa'
```

Comment:
I just realized - these tests are independent tasks and shouldn't have been in this function at all.